sdk/devcenter/developer-devcenter-rest/package.json

```diff
@@ -60,8 +60,8 @@
     "@azure/core-rest-pipeline": "^1.5.0",
     "@azure/logger": "^1.0.0",
     "tslib": "^2.6.2",
-    "@azure/core-lro": "^2.5.4",
-    "@azure/abort-controller": "^1.0.0",
+    "@azure/core-lro": "3.0.0-beta.1",
```

Comment:
Changes look good. We probably don't want to release a GA version of devcenter with a dependency on a beta version of core-lro? @joheredi When will we GA core-lro?

Reply:
Yeah, we will not include the beta core-lro in GAed SDK. @drielenr May I know what's your plan to release new version?

Reply:
We should release as soon as this gets ready, and we get all the approvals